templates/frontend/components/breadcrumbs.tpl

```diff
@@ -23,13 +23,16 @@
 			<span class="separator">{translate key="navigation.breadcrumbSeparator"}</span>
 		</li>
 		<li class="current">
-			<h1>
+			{capture name="currentTitleH1"}
 				{if $currentTitleKey}
 					{translate key=$currentTitleKey}
 				{else}
 					{$currentTitle|escape}
 				{/if}
-			</h1>
```

Comment:
Removing heading from breadcrumb trail is perfect. Hoewever, given we are fixing this issue for headings on breadcrumbs and our accessibility report has noted it, could be possible also add a link to the current page on this element with parameter `aria-current="page"`? 
This would improve accessibility while screen readers should announce this link is the current page in the navigation.

Reference: https://www.w3.org/TR/wai-aria-practices/examples/breadcrumb/index.html#ex_label